include/windows/osd.h

```diff
@@ -53,17 +53,43 @@ struct iovec
 
 typedef int pid_t;
 
-static int clock_gettime(int which_clock, struct timespec* tp)
+#define exp7		  10000000i64	/* 1E+7 */
+#define exp9		1000000000i64	/* 1E+9 */
```

Comment:
Sure. I gave more clear names. Moreover, I've removed the usage of high-precision Windows timing and use instead `GetSystemTimeAsFileTime` that is more stable when the process switches from one core to another core (seems it was a cornerstone of the instability of Windows time computation). Because, fortunately, our fabtests doesn't require high-precision time computation we can avoid usage of HW-dependent procedure and use more simple solution.